Cargo.toml

```diff
@@ -7,32 +9,39 @@ members = [
 	"clap_mangen",
 ]
 
+[workspace.package]
+categories = ["command-line-interface"]
+edition = "2018"
+include = [
+    "build.rs",
+    "src/**/*",
+    "Cargo.toml",
+    "LICENSE*",
+    "README.md",
+    "benches/**/*",
+    "examples/**/*"
+]
+license = "MIT OR Apache-2.0"
+readme = "README.md"
+
 [package]
 name = "clap"
 version = "3.1.18"
```

Comment:
I wonder about clap and clap_derive using a shared version

Reply:
Are they always the same?

Reply:
Yes, they are always the same.  This currently is getting enforced in `cargo-release` via the `shared-version = true` setting.  I mostly did that to dog food the setting but now that we use a `=` version req, I think its helps clarify the relationship.